app/models/section.rb

```diff
@@ -152,7 +158,8 @@ def mark_as_exported!(exported_at = Time.zone.now)
   end
 
   def persisted?
-    editions.any?(&:persisted?)
+    previous_edition && previous_edition.persisted? ||
+      latest_edition.persisted?
```

Comment:
Minor: I would wrap the first two conditions in parentheses rather than relying on operator precedence.